glue/viewers/profile/state.py

```diff
@@ -280,7 +316,13 @@ def update_profile(self, update_limits=True):
             data = self.layer
             subset_state = None
 
-        profile_values = data.compute_statistic(self.viewer_state.function, self.attribute, axis=axes, subset_state=subset_state)
+        if self.viewer_state.function == 'slice':
+            data_slice = list(copy.deepcopy(self.viewer_state.slices))
```

Comment:
If slices is a list/tuple you can just do ``data_slice = self.viewer_state.slices[:]``

Reply:
Just to be clear, ``[:]`` does to a copy for a list/tuple (unlike Numpy arrays). But I'm ok with leaving it as is as it is more explicit.

Reply:
Okay... Agree that it is better to be more explicit here.